src/statement.d

```diff
@@ -241,491 +236,13 @@ extern (C++) abstract class Statement : RootObject
     }
 
     /* ============================================== */
-    /** Only valid after semantic analysis
```

Comment:
FWIW at some point there was also the movement to expose `dmd` as library and there only `**` comments would be publicly documented.